File: ios/Plugin/KlarnaKcoConfig.swift
import Foundation
import UIKit
import KlarnaMobileSDK
import NotificationBannerSwift

public struct KlarnaKcoConfig {
    var iosReturnUrl = URL.init(string: "")
    var handleValidationErrors: Bool = false
    var handleEPM: Bool = false
    var region: KlarnaRegion = KlarnaRegion.eu
    var environment: KlarnaEnvironment = KlarnaEnvironment.playground
    var loggingLevel: KlarnaLoggingLevel = KlarnaLoggingLevel.off
    var theme: KlarnaTheme = KlarnaTheme.light
}

File: android-oldish/src/main/java/com/capacitor/community/klarna/kco/KlarnaKcoConfig.kt
package com.capacitor.community.klarna.kco

import com.klarna.mobile.sdk.api.KlarnaEnvironment
import com.klarna.mobile.sdk.api.KlarnaLoggingLevel
import com.klarna.mobile.sdk.api.KlarnaRegion

class KlarnaKcoConfig {
    var androidReturnUrl = ""
    var handleValidationErrors = false
    var handleEPM = false
    var loggingLevel = KlarnaLoggingLevel.Off
    var region = KlarnaRegion.EU
    var environment = KlarnaEnvironment.PLAYGROUND
}